src/main/shadow/cljs/devtools/server/web.clj

```diff
@@ -42,8 +42,9 @@
       [:title (-> (io/file ".")
                   (.getCanonicalFile)
                   (.getName))]
-      [:link {:rel "stylesheet" :href "/css/main.css"}]
-      [:link {:rel "stylesheet" :href "/css/tailwind.min.css"}]
+      #_[:link {:rel "stylesheet" :href "/css/main.css"}]
+      #_[:link {:rel "stylesheet" :href "/css/tailwind.min.css"}]
+      [:link {:rel "stylesheet" :href "https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css"}]
```

Comment:
Don't pull from CDN. You can download the file the file and put it into the `src/ui-release` folder the current one is in.

Reply:
What is the main motivation here? To work offline?

Reply:
unpkg could be slow from China, or some other locations

Reply:
Don't need to talk to the internet when you can get it fast locally yeah.